### ISSUE TITLE
Add std.experimental.logger 

### ISSUE THREAD

**Issue by andre2007** [NCrashed/daemonize#16]
Phobos comes now with std.experimental.logger. Maybe this could be set as default logger?

**Comment by NCrashed** [comment]
Yep, it will be better for default option. So, if you make a PR with the improvement don't forget to add yourself to list of contributors in dub.json.
